tests/manage/z_cluster/nodes/test_nodes_maintenance.py

```diff
@@ -49,6 +53,20 @@ def init_sanity(self):
         """
         self.sanity_helpers = Sanity()
 
+    @pytest.fixture(autouse=True)
+    def health_checker(self):
+        """
+        Check Ceph health
+
+        """
+        try:
```

Comment:
I would add:
```
logger.info('Checking Ceph health. HEALTH_OK status is a prerequisite for this test.')
```
at the beginning of the fixture so it's clear from log why it fails if Ceph is not healthy.